docs/tools/offline/cli-test-debug.md

```diff
@@ -2,23 +2,30 @@
 
 Use the `mbed test` command to compile and run tests.
 
+There are two testing frameworks: Greentea and Icetea. Greentea provides tests designed for driver porting and target verification. Icetea provides and manages tests for multiple devices at the same time. Devices being tested are referred to as devices under test (DUTs). For example, you can test the network setup for a server and multiple clients, simultaneously controlling them from the test environment.
+
 The arguments to `test` are:
-* `-m <MCU>` to select a target for the compilation. If `detect` or `auto` parameter is passed, then Mbed CLI will attempt to detect the connected target and compile against it.
-* `-t <TOOLCHAIN>` to select a toolchain (of those defined in `mbed_settings.py`, see above), where `toolchain` can be either `ARM` (Arm Compiler 5), `GCC_ARM` (GNU Arm Embedded), or `IAR` (IAR Embedded Workbench for Arm).
-* `--compile-list` to list all the tests that can be built.
-* `--run-list` to list all the tests that can be run (they must be built first).
-* `--compile` to only compile the tests.
-* `--run` to only run the tests.
-* `-n <TESTS_BY_NAME>` to limit the tests built or run to a comma separated list (ex. test1,test2,test3).
-* `--source <SOURCE>` to select the source directory. Default is `.` (the current directory). You can specify multiple source locations, even outside the program tree.
-* `--build <BUILD>` to select the build directory. Default: `BUILD/` inside your program.
-* `--profile <PATH_TO_BUILD_PROFILE>` to select a path to a build profile configuration file. Example: `mbed-os/tools/profiles/debug.json`.
-* `-c or --clean` to clean the build directory before compiling.
-* `--test-spec <TEST_SPEC>` to set the path for the test spec file used when building and running tests (the default path is the build directory).
-* `-v` or `--verbose` for verbose diagnostic output.
-* `-vv` or `--very_verbose` for very verbose diagnostic output.
-
-Invoke `mbed test`:
+* `-m <MCU>`: to select a target for the compilation. If the `detect` or `auto` parameter is passed, then Mbed CLI will attempt to detect the connected target and compile against it.
```

Comment:
I need to review these in Atom's Markdown preview to make sure these render as a list and not on a single line.

Reply:
Atom is a different flavour of MD from our engine. As best practice we always add an empty line.